uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <!-- TODO(wraftus): Play around with scan_height and range_min -->
+    <arg name="scan_height"      default="10"/> 
+    <arg name="range_min"        default="0.45"/>
+    
+    <!-- TODO(wraftus): Make names of ZEDs more descriptive depending on location on robot-->
+    <!-- First ZED camera -->
+    <group ns="zed0">
+        <node name="zed_wrapper" pkg="zed_wrapper" type="zed_wrapper_node" output="screen" required="true">
+            <param name="zed_id"                value="0" />
+            <param name="camera_model"          value="0" />
+            <param name="verbose"               value="false" />
+            
+            <param name="resolution"            value="2" />
+            <param name="frame_rate"            value="30" />
+            <param name="gain"                  value="100" />
+            <param name="exposure"              value="100" />
+            <param name="auto_exposure"         value="true" />
+            <param name="mat_resize_factor"     value="1.0" />
+            
+            <param name="quality"               value="1" />
+            <param name="sensing_mode"          value="0" />
+            <param name="openni_depth_mode"     value="0" />
+            <param name="confidence"            value="100" />
+            <param name="max_depth"             value="10.0" />
+            <param name="depth_stabilization"   value="1" />
+            
+            <param name="pose_smoothing"        value="false" />
+            <param name="spatial_memory"        value="false" />
+            <param name="floor_alignment"       value="false" />
+            <param name="camera_flip"           value="false" />
+            <param name="publish_tf"            value="true" />
+            <param name="publish_map_tf"        value="true" />
+            <param name="init_odom_with_first_valid_pose"   value="true" />
+            <param name="publish_pose_covariance"   value="false" />
+
+            <param name="depth_topic"           value="depth_registered" />
+            <param name="base_frame"            value="zed_camera_center" />
```

Comment:
`zed_camera_center` is the default value, so this line is not needed

Reply:
thats what I thought too, but for some reason zed_camera_center isn't actually the default. Says it cant find reference frame unless I set it

Reply:
You're right - I just looked at the code, `camera_frame` is default `zed_camera_center` but `base_frame` is default `base_link`. Looks like more documentation errors on StereoLabs' part.